src/ui/componentSection/index.ts

```diff
@@ -28,6 +28,7 @@ interface cardComponentRender {
   props: {
     title?: string,
     subtitle?: string,
-    locationId: string,
+    locationId?: string,
+    afterHeaderLocationId?: string,
```

Comment:
What does this mean exactly? A whole iFrame after a card or a iFrame inside the card footer?

If it is a whole iFrame after a Card I would suggest to add a new component for this instead. Like this:

```
interface iFrameComponentRender {
  component: 'iframe',
  props: {
    locationId: string 
  }
}
```